### PR TITLE
Add store footer

### PR DESCRIPTION
**Pull request by sfillm07**
I think I am up to date with the master.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta name="description" content="Veteran-founded and led, Operation Code is a 501(c)(3) non-profit whose mission is to aid military, vets and their families learn coding and web technologies.">
   <meta name="keywords" content="operation code, veterans, software development, learn to code, keep coding, veteran coders, senate veterans, house veterans, gi bill, new gi bill, operationcode.org, #operationcode, military vets, vets">
   <meta name="author" content="Veteran Coders, for Veterans Coders">
+  <meta property="og:image" content="https://static1.squarespace.com/static/586d87de725e2585df6e8448/t/5870110d6b8f5b61fdd8ee05/1483739408996/?format=500w" />
```
Comment (rickr) [inline app/views/layouts/application.html.erb]: What's this line for?

Comment (sfillm07) [inline app/views/layouts/application.html.erb]: That was from a previous pull request I sent a few weeks ago.  That changed the image used on the FB posts.  When I sent you the previous pull requests I hadn't used a branch, I was making all the changes in my master.

Comment (rickr) [inline app/views/layouts/application.html.erb]: Hrm. Do you know what pull request it was? I'm not seeing it in our master branch.

**Comment by sfillm07** [comment]
I sent the pull request almost a month ago, not sure if it ever got merged.  That is for issue# 372... 

    On Tuesday, February 21, 2017 9:13 AM, Rick <notifications@github.com> wrote:
 

 @rickr commented on this pull request.In app/views/layouts/application.html.erb:> @@ -7,6 +7,7 @@
   <meta name="description" content="Veteran-founded and led, Operation Code is a 501(c)(3) non-profit whose mission is to aid military, vets and their families learn coding and web technologies.">
   <meta name="keywords" content="operation code, veterans, software development, learn to code, keep coding, veteran coders, senate veterans, house veterans, gi bill, new gi bill, operationcode.org, #operationcode, military vets, vets">
   <meta name="author" content="Veteran Coders, for Veterans Coders">
+  <meta property="og:image" content="https://static1.squarespace.com/static/586d87de725e2585df6e8448/t/5870110d6b8f5b61fdd8ee05/1483739408996/?format=500w" />
Hrm. Do you know what pull request it was? I'm not seeing it in our master branch.—

**Comment by rickr** [comment]
Still haven't time to get at least 1 item in the store.

**Pull request merged**